api/spec/requests/spree/api/promotion_application_spec.rb

```diff
@@ -20,6 +20,7 @@ module Spree::Api
 
       it "can apply a coupon code to the order" do
```

Comment:
this tests look like duplicated of what we have [here](https://github.com/nebulab/solidus/pull/19/files#diff-72468e52a59aedfe8a3443597f1a6f3aR856), right?  If yes, maybe we can do another PR to remove duplication?

Reply:
Noted :ok_hand: